server/core/store.go

```diff
@@ -305,6 +305,16 @@ func (s *StoreInfo) regionScoreV2(delta int64, deviation int, lowSpaceRatio floa
 	A := float64(float64(s.GetAvgAvailable())-float64(deviation)*float64(s.GetAvailableDeviation())) / gb
 	C := float64(s.GetCapacity()) / gb
 	R := float64(s.GetRegionSize() + delta)
+	if R < 0 {
```

Comment:
em, why is `deviation` not removed? (Line 305)

Reply:
I think the stragy can config before, so it could roll back after it online